templates/procuration/landing.html.twig

```diff
@@ -42,7 +42,7 @@
                             Un(e) habitant(e) de ma ville va voter en mon nom.
                         </div>
 
-                        <a href="{{ path('app_procuration_index') }}" class="btn btn--blue">
+                        <a href="{{ path('app_procuration_choose_election', { action: 'demande' }) }}" class="btn btn--blue">
```

Comment:
I think you can use the `ACTION_REQUEST` const here ?

Reply:
Good catch, fixed in https://github.com/EnMarche/en-marche.fr/pull/1948/commits/81f9332262da308744868890e8828cf30cf4fce4.